tests/basic_env.py

```diff
@@ -48,6 +48,12 @@ class TestDirFixture(object):
     )
     UNICODE = "тест"
     UNICODE_CONTENTS = "проверка"
+    REGULAR_DIR = "lib"
```

Comment:
From TestDirFixture perspective, all files/dirs(DATA, DATA_DIR, FOO, BAR, etc) are regular and not tracked by dvc. You probably want to modify erepo fixture to have those files, not this global fixture.

Reply:
Why exactly do those files need to be tracked by dvc? I thought we were testing the retrieval of a normal file that is tracked by Git.

Reply:
> Why exactly do those files need to be tracked by dvc? I thought we were testing the retrieval of a normal file that is tracked by Git.

They don't, that is why "regular file/dir" terminology doesn't make sense here 🙂These files(FOO, BAR, etc) are `dvc add`ed in erepo fixture, and this base fixture you are modifying doesn't do any assumptions like that. That is why you should modify erepo fixture(or somewhere on top of it) to add files/dirs that are not `dvc add`ed, instead of modifying the unrelated base fixture and breaking unrelated tests.

Reply:
We should stop creating those gigantic fixtures. I suggest making them more granular so that you only create whatever you need for your tests. Now it is lots of unneeded work for most tests.

Reply:
And we can start by not adding things to `erepo` not to `TestDirFixture`.

Reply:
I agree with @Suor "One structure for all" also makes debugging harder.